packages/fxa-auth-server/lib/routes/subscriptions.ts

```diff
@@ -1190,6 +1202,15 @@ class DirectStripeRoutes {
       return [];
     }
 
+    if (!customer.subscriptions) {
+      throw error.internalValidationError(
+        'getSubscriptions',
+        {
+          customerId: customer.id,
+        },
+        'Customer did not have any subscriptions.'
```

Comment:
Miniscule nit, but: At first I thought, isn't it expected for a customer to not have any subscription if they were all deleted? But this is actually the the customer object is missing a subscriptions property altogether - not even an empty list. Not sure if that error would confuse us later.